src/control.c

```diff
@@ -9,14 +9,14 @@
  */
 static volatile float linear_acceleration = 5.;
```

Comment:
why these changes on this commit?

Reply:
Just for now, while the robot keeps crashing into walls like a drunken robot... I will remove it before merging into master. I hope...